src/interfaces/stepper-interfaces/AddPeopleAndRole/people-and-roles-resource-interface.ts

```diff
@@ -14,7 +14,8 @@ export interface PeopleAndRolesResourceIF {
     helpText: Array<string>
   }
   blurb2?: string | Array<string>
-  addOrganization?: string
+  addIncorporator?: boolean
+  addOrganization?: boolean
```

Comment:
`addOrganization` is now a boolean since it's either not displayed, or the same string for everyone.

Reply:
What do you think of changing the name of `addOrganization` to indicate it's a Boolean?  I've notice elsewhere in the code we often use `isSomething` for Booleans.

Reply:
That a good idea. OK if I change this next PR?

I will note that there are now 3 flags in here with similar names and function.